code/__HELPERS/time.dm

```diff
@@ -30,7 +30,7 @@
 /proc/time_stamp()
 	var/date_portion = time2text(world.timeofday, "YYYY-MM-DD")
 	var/time_portion = time2text(world.timeofday, "hh:mm:ss")
-	return "[date_portion]T[time_portion]"
+	return "[date_portion] T[time_portion]"
```

Comment:
This proc is used for getting timestamps inside of files written to disk. Do not change its format.